cosmo_tester/test_suites/cli/linux_test.py

```diff
@@ -44,11 +50,180 @@ def test_cli_install_flow_linux(linux_cli_tester, logger):
+                [hashlib.md5(open(f, 'rb').read()).hexdigest() for f in
+                 files if not ('journalctl' in f.name or
+                               'access' in f.name or
+                               'patroni' in f.name or
+                               'startup_check' in f.name)])
+            assert set(log_hashes) == set(log_hashes_local)
+
+    logger.info('Testing `cfy logs backup`')
+    cli_host.run_command('{cfy} logs backup --verbose'.format(
+        cfy=paths['cfy']))
+    output = nodes[0].run_command('ls /var/log').stdout
+    assert 'cloudify-manager-logs_' in output
+
+    logger.info('Testing `cfy logs purge`')
+    cli_host.run_command('{cfy} logs purge --force'.format(cfy=paths['cfy']))
+
+    # Verify that each file under /var/log/cloudify is size zero
+    nodes[0].run_command(
+        'for f in $(sudo find /var/log/cloudify -type f -name "*") ; '
+        'do if $(test -s $f) ; then echo $f && exit 1 ; fi ; done'
+    )
+
+
+@pytest.fixture(
+    scope='module',
+    params=get_linux_image_settings())
+def linux_cluster_cli_tester(three_nodes_cluster, request, ssh_key,     # NOQA
+                             module_tmpdir, test_config, logger,
+                             install_dev_tools=True):
+
+    cli_instance = get_image('centos', test_config)
+    image, username = get_image_and_username(request.param[0], test_config)
+    cli_instance.image_name = image
+    cli_instance.username = username
+
+    cli_hosts = Hosts(
+        ssh_key, module_tmpdir,
+        test_config, logger, request, instances=[cli_instance])
+
+    passed = True
+    try:
+        cli_hosts.create()
+
+        url_key = request.param[1]
+        pkg_type = request.param[2]
+        cli_host = cli_hosts.instances[0]
+
+        _install_cli_client(cli_host, logger, url_key, ssh_key,
+                            pkg_type, test_config)
+
+        logger.info('Copying agent ssh key and CA cert to CLI host')
+        remote_ssh_key_path = '/tmp/cli_test_ssh_key.pem'
+        cli_host.put_remote_file(
+            remote_path=remote_ssh_key_path,
+            local_path=ssh_key.private_key_path,
+        )
+        remote_ca_cert_path = '/tmp/cli_test_ca.cert'
+        cli_host.put_remote_file(
+            remote_path=remote_ca_cert_path,
+            local_path='{}/ca.cert'.format(ssh_key.tmpdir),
+        )
+
+        logger.info('Using manager')
+        cli_host.run_command(
+            'cfy profiles use {ip} -u admin -p admin -t '
+            'default_tenant --ssh-user {ssh_user} --ssh-key {ssh_key} '
+            '-c {rest_cert}'.format(
+                ip=three_nodes_cluster[0].ip_address,
+                ssh_user=three_nodes_cluster[0].username,
+                ssh_key=remote_ssh_key_path,
+                rest_cert=remote_ca_cert_path,
+            )
+        )
+
+        # monkey with the profile so cluster nodes have an external ip,
+        # because the CLI host and the managers don't share an internal network
```

Comment:
They should all be on the same network because only one network is deployed in any tests except multi-network ones. What are you seeing if you don't do this?